mulog-kinesis/test/com/brunobonacci/mulog/publishers/test_publisher.clj

```diff
@@ -0,0 +1,56 @@
+(ns com.brunobonacci.mulog.publishers.test-publisher
```

Comment:
please rename this to `com.brunobonacci.mulog.publishers.kinesis.test-publisher`
since is interesting only for kinesis testing